schemas/common/descriptors/relationship/descriptorOneToOne.schema.json

```diff
@@ -0,0 +1,68 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/common/descriptors/descriptorOneToOne",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "One to One Relationship Descriptor",
+  "meta:extensible": false,
+  "meta:extends": [
+    "https://ns.adobe.com/xdm/common/desciptors/schemadescriptor#/definitions/descriptor",
+    "https://ns.adobe.com/xdm/common/descriptors/relationshipdescriptor#/definitions/relationshipdescriptor"
+  ],
+  "meta:abstract": false,
+  "type": "object",
+  "description": "Describes one to one relationship descriptor.",
+  "definitions": {
+    "descriptorOneToOne": {
+      "properties":{
+        "@type": {
+          "type": "string",
+          "const": "https://ns.adobe.com/xdm/common/descriptors/descriptorOneToOne",
+          "meta:enum": {"https://ns.adobe.com/xdm/common/descriptors/descriptorOneToOne": "xdm:descriptorOneToOne"}
+        }
+      },
+      "required": [
+        "xdm:destinationSchema",
+        "xdm:sourceSchema"
+      ]
+    }
+  },
+  "allOf": [
+    {
+      "$ref": "https://ns.adobe.com/xdm/common/desciptors/schemadescriptor#/definitions/descriptor"
+    },
+    {
+      "$ref": "https://ns.adobe.com/xdm/common/descriptors/relationshipdescriptor#/definitions/relationshipdescriptor"
+    },
+    {
+      "$ref": "#/definitions/descriptorOneToOne"
+    },
+    {
+      "oneOf": [
```

Comment:
why not add this to the base descriptor itself? requiring a label or a src-property should be relevant to all relationships.